beater/api/mux.go

```diff
@@ -47,7 +47,7 @@ const (
 	RootPath = "/"
 
 	// ConfigAgent defines the path to query for agent config management
-	ConfigAgent = "/config/v1/agents"
+	AgentConfigPath = "/config/v1/agents"
```

Comment:
i only changed it because of readability, we always call this "agent config" and not "config agent" (which doesn't read very naturally)... the actual string value is not so important